src/cephadm/cephadm

```diff
@@ -2054,19 +2093,11 @@ def command_deploy():
                 raise Error("{} deployment requires config-json which must "
                             "contain arg for {}".format(daemon_type.capitalize(), ', '.join(required_args)))
 
-        if daemon_type == 'prometheus':
-            uid, gid = extract_uid_gid(file_path='/etc/prometheus')
-        elif daemon_type == 'node-exporter':
-            uid, gid = 65534, 65534
-        elif daemon_type == 'grafana':
-            uid, gid = extract_uid_gid(file_path='/var/lib/grafana')
-        elif daemon_type == 'alertmanager':
-            uid, gid = extract_uid_gid(file_path='/alertmanager')
-        else:
-            raise Error("{} not implemented yet".format(daemon_type))
+        uid, gid = uid_gid_monitoring(daemon_type)
```

Comment:
nit: extract_uid_gid_monitoring() would be consistent with existing function naming

Reply:
changed.